loadtest/user/userentity/actions.go

```diff
@@ -134,23 +123,29 @@ func (ue *UserEntity) Login() error {
 	if err != nil {
 		return err
 	}
+	var loggedUser *model.User
 
 	switch ue.config.AuthenticationType {
 	case AuthenticationTypeOpenID:
 		if err := ue.authOpenID(authOpenIDLogin); err != nil {
 			return fmt.Errorf("error while logging in using OpenID: %w", err)
 		}
+
+		loggedUser, _, err = ue.client.GetUserByUsername(context.Background(), user.Username, "")
```

Comment:
Ah okay, so `Signup` already had the right code. We just hadn't applied it to `Login`? That feels a bit weird.

Reply:
Yeah, in my changes (thinking it wasn't working at that particular time because of a Keycloak issue) I forgot that `Login` should store the user the same way as `SignUp` does.